package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "notifications",
+  "version": "2.2.0",
+  "description": "This app provides a backend and frontend for the notification API available in Nextcloud.",
+  "main": "init.js",
+  "directories": {
+    "lib": "lib",
+    "test": "tests"
+  },
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack --progress --hot --config js-src/webpack.config.js --watch",
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules --config js-src/webpack.config.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nextcloud/notifications.git"
+  },
+  "author": "Joas Schilling",
+  "license": "AGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/nextcloud/notifications/issues"
+  },
+  "homepage": "https://github.com/nextcloud/notifications#readme",
+  "dependencies": {
+    "vue": "^2.5.13"
+  },
+  "devDependencies": {
+    "babel-core": "^6.26.0",
```

Comment:
not sure if you actually need/use babel. Looks like it's all ES5 and thus can be run in all our supported browsers.